tests/cursor/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -121,8 +120,8 @@ const y = 5
 
 const z = 9
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-const y = 5;
-<|>
+const y = 5<|>;
+
```

Comment:
The cursor was on the line after the 5, you don't want it to become in the same line as the 5